lib/rectangles.js

```diff
@@ -1,17 +1,17 @@
-function area(l, w) {
-  return l * w;
+function area(rectangle) {
+  return rectangle.length * rectangle.width;
```

Comment:
A suggestion here - what about creating a Rectangle class similar to your Triangle class that would have its own `area` method? This way, rather than designing an `area` method in the global scope that has to know about the structure of a rectangle coming in, it would be a method specifically made for rectangles and would have access to the rectangle's length and width properties.